extensions/amp-story/1.0/amp-story.js

```diff
@@ -1322,6 +1328,20 @@ export class AmpStory extends AMP.BaseElement {
     this.activePage_.setState(pageState);
   }
 
+  /**
+   * Reacts to paused state updates.
+   * @param {boolean} open
+   * @private
+   */
+  onSidebarStateUpdate_(open) {
+    const actions = Services.actionServiceForDoc(this.getAmpDoc());
+    if (open && this.sidebar_) {
+      actions.execute(this.sidebar_, 'open', /* args */ null, /* source */ null,
+          /* caller */ null, /* event */ null, ActionTrust.HIGH);
+      this.storeService_.dispatch(Action.TOGGLE_SIDEBAR, false);
```

Comment:
Could we set this to false only when the sidebar closes? That'd be helpful if we ever need to know if the sidebar is open.
Additionally, you could change the `PAUSED_STATE` from the `TOGGLE_SIDEBAR` action. There's an example [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/amp-story-store-service.js#L152-L162) if you need it :)